src/main/java/org/radarcns/management/web/rest/errors/ExceptionTranslator.java

```diff
@@ -79,7 +79,7 @@ public ParameterizedErrorVM processParameterizedConflict(CustomConflictException
     }
 
     @ExceptionHandler(AccessDeniedException.class)
-    @ResponseStatus(HttpStatus.FORBIDDEN)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
Forbidden seems appropriate. According to the `AccessDeniedException` javadoc:
> Thrown if an Authentication object does not hold a required authority.

So the person is authenticated but forbidden. See also: http://www.dirv.me/blog/2011/07/18/understanding-403-forbidden/index.html, otherwise a `WWW-Authenticate` header should also be added.

Reply:
Indeed, reverting this change